api/src/test/java/org/openmrs/util/OpenmrsUtilUnitTest.java

```diff
@@ -478,4 +479,35 @@ public void getRuntimePropertiesFilePathName_shouldReturnNullIfPropertyFileNotEx
 		assertNull(OpenmrsUtil.getRuntimePropertiesFilePathName("app-openmrs"));
 	}
 
+	@Test
+	public void getDeclaredFields_shouldReturnSetOfSizeZeroForPrimitiveTypeArrayTypeOrVoid() {
+
+		assertThat(OpenmrsUtil.getDeclaredFields(int.class).size(), is(0));
+		assertThat(OpenmrsUtil.getDeclaredFields(int[].class).size(), is(0));
+		assertThat(OpenmrsUtil.getDeclaredFields(void.class).size(), is(0));
+	}
+
+	@Test
+	public void convertToInteger_shouldThrowExceptionIfLongValueOutOfIntegerRange() {
+
```

Comment:
you can remove  the spaces immediate after the methods, check your other methods too

Reply:
@HerbertYiga every method in this test file has this space, should I remove it?

Reply:
yes

Reply:
@HerbertYiga I tried but the space is coming after reformatting my changes can you check if its not the required code-style? since it is being followed by every other method?